src/Arcus.Scripting.LogicApps/Scripts/Cancel-AzLogicAppRuns.ps1

```diff
@@ -0,0 +1,18 @@
+param(
```

Comment:
Super great work on this, Pim! Good formatting for maximum readability, great colorful logging messages to maximize defect localization, and exception handling in case something is wrong. 🥇 .

Reply:
Thanks!